docker-image/v1.15/arm64/debian-opensearch/Gemfile

```diff
@@ -15,7 +15,7 @@ gem "fluent-plugin-record-modifier", "~> 2.1.0"
 gem "fluent-plugin-detect-exceptions", "~> 0.0.13"
 gem "fluent-plugin-rewrite-tag-filter", "~> 2.4.0"
 gem "fluent-plugin-parser-cri", "~> 0.1.0"
-gem "fluent-plugin-opensearch", "~> 1.0.6"
+gem "fluent-plugin-opensearch", "~> 1.0.8"
```

Comment:
Sure, I'll rollback that change.

Reply:
Just to make sure I have the whole picture, my understanding is that you update the version of the package in the template (templates/gemfile.erb) and the makefile will render the gemfiles for the dockerfiles, but the version in the template still needs to be manually updated correct?

I'm asking, because I'm noticing that the latest version of master in the template is at 1.0.6: https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/templates/Gemfile.erb#L30

However, version 1.0.8 is out: https://github.com/fluent/fluent-plugin-opensearch/releases/tag/v1.0.8

Reply:
> Just to make sure I have the whole picture, my understanding is that you update the version of the package in the template (templates/gemfile.erb) and the makefile will render the gemfiles for the dockerfiles, but the version in the template still needs to be manually updated correct?

That's right, templates/Gemfile.erb needs to be updated manually.
In addition, "~> 1.0.6" means `1.0.6 or later in 1.0.x series`.
So that you don't always need to update it.

Reply:
I see. Thanks for the clarification.

Reply:
Done